Pod/Classes/NJHNavigationBarBackgroundSelectionView.h

```diff
@@ -0,0 +1,67 @@
+//
+//  NJHNavigationBarBackgroundSelectionView.h
+//  Pods
+//
+//  Created by Hakon Hanesand on 7/29/15.
+//
+//
+
+@class NJHNavigationBarSelectionView;
+
+@import UIKit;
+
+@protocol NJHNavigationBarBackgroundSelectionViewDelegate <NSObject>
+
+/**
+ *  Called when the user presses the NJHNavigationBarBackgroundSelectionView
+ *
+ *  @param point The point the user pressed
+ */
+- (void)userDidTapBackgroundSelectionViewAtLocation:(CGPoint)point;
+
+@end
+
+@interface NJHNavigationBarBackgroundSelectionView : UIControl
+
+/**
+ *  Encapsulates the process of instantiating this view from it nib
+ *
+ *  @return An instance of this view with its views loaded from the appropriate nib
+ */
++ (instancetype)instance;
+
+/**
+ *  The delegate of this view that is notified if the user taps the navigation bar at a certain position, usually
+ *  to indicate that the user wishes to navigate to that section
+ */
+@property (nonatomic, weak) id <NJHNavigationBarBackgroundSelectionViewDelegate> delegate;
+
+/**
+ *  The view that slides around on top of this background view to signify what view controller is selected
+ */
+@property (weak, nonatomic) IBOutlet UIView *selectorView;
+
+/**
+ *  The titles for the labels displayed under the selection view, normally the titles of the view controllers
+ *  that can be selected
+ */
+@property (nonatomic) NSArray *labelTitles;
+
+/**
+ *  The font for the title labels. Default font is system font of size 10
+ */
+@property (nonatomic) UIFont *fontForLabels;
```

Comment:
If you specify the type and what is consists of, would it not be suitable to call it labelFonts? Is there a reasoning for this naming convention?

I would normally name it, 
UIButton *settingsButton and NSString *titleString etc :)